test/visual-diff/visual-tests

```diff
@@ -210,26 +210,25 @@
       "loading_complete_selectors": ["video.i-amphtml-replaced-content"]
     },
     {
-      // TODO(#27456, @ampproject/wg-ui-and-a11y): see https://percy.io/ampproject/amphtml/builds/4656426
-      "flaky": true,
       "url": "examples/visual-tests/amp-sidebar/amp-sidebar.amp.html",
       "name": "amp-sidebar",
       "viewport": {"width": 400, "height": 800},
       "interactive_tests": "examples/visual-tests/amp-sidebar/amp-sidebar.js",
+      "loading_incomplete_selectors": ["amp-sidebar.i-amphtml-element"]
```

Comment:
should these be `loading_complete_selectors`?

Reply:
`loading_complete_selectors` expects the element to be present and visible -- when sidebar is built it is present and invisible. `incomplete` is described: `CSS selectors for elements that may initially appear on the page, but must eventually be removed from it or become invisible`. 

Not sure if this invariant is satisfied at the start because it is not present or later one when it becomes present and invisible. 🤔 But the other one is definitely not suitable because it causes the test to hang.

/cc @danielrozenberg

Reply:
Looks like `loading_incomplete_selectors` [uses `verifySelectorsInvisible`](https://github.com/ampproject/amphtml/blob/9d7d8d5fc215113b4119097dcdece8e3190f9d80/build-system/tasks/visual-diff/index.js#L539), so that makes sense. Thanks for clarifying. 

I wonder if the semantics would be clearer if the name was more consistent?
@danielrozenberg 🙂

Reply:
We can definitely change those if you think the current name is bad